src/state/megaStore.tsx

```diff
@@ -192,30 +191,18 @@ export const Provider: ParentComponent = (props) => {
                 // If we get this far then we don't need the password anymore
                 setState({ needs_password: false });
 
-                // Subscription stuff. Skip if it's not already in localstorage
-                let subscription_timestamp: number | undefined = undefined;
-                const stored_subscription_timestamp = localStorage.getItem(
-                    "subscription_timestamp"
-                );
-                // If we have a stored timestamp, check if it's still valid
-                if (stored_subscription_timestamp) {
-                    try {
-                        const timestamp =
-                            await mutinyWallet?.check_subscribed();
-
-                        // Check that timestamp is a number
-                        if (!timestamp || isNaN(Number(timestamp))) {
-                            throw new Error("Timestamp is not a number");
-                        }
-
-                        subscription_timestamp = Number(timestamp);
-                        localStorage.setItem(
-                            "subscription_timestamp",
-                            timestamp.toString()
-                        );
-                    } catch (e) {
-                        console.error(e);
+                // Check if we're subscribed and update the timestamp
+                try {
+                    const timestamp = await mutinyWallet?.check_subscribed();
+
+                    // Check that timestamp is a number
+                    if (!timestamp || isNaN(Number(timestamp))) {
+                        throw new Error("Timestamp is not a number");
```

Comment:
Would be nice to not throw the error when `timestamp` is undefined so we dont get the scary error log on every new wallet created

Reply:
fixed, can you double check my booleans?